XLS-0070d-credentials/README.md

```diff
@@ -74,18 +74,21 @@ This ledger object is an on-chain representation of a credential.
 
 This object costs one owner reserve for either the issuer or the subject of the credential, depending on whether the subject has accepted it.
 
+The `Credential` object will live in both the `Subject` and `Issuer`'s owner directories (similar to a trustline or escrow).
+
 ### 2.1. Fields
 
 | Field Name | Required? | JSON Type | Internal Type | Description |
 |------------|-----------|-----------|---------------|-------------|
-|`LedgerEntryType`| ✔️|`string`|`UInt16`|The ledger object's type (`Credential`).|
-|`Flags`| ✔️|`number`|`UInt32`|Flag values associated with this object.|
-|`Subject`| ✔️|`string`|`AccountID`|The account that the credential is for.|
-|`Issuer`| ✔️|`string`|`AccountID`|The issuer of the credential.|
-|`CredentialType`| ✔️|`string`|`Blob`|A (hex-encoded) value to identify the type of credential from the issuer.|
+|`LedgerEntryType`|✔️|`string`|`UInt16`|The ledger object's type (`Credential`).|
+|`Flags`|✔️|`number`|`UInt32`|Flag values associated with this object.|
+|`Subject`|✔️|`string`|`AccountID`|The account that the credential is for.|
+|`Issuer`|✔️|`string`|`AccountID`|The issuer of the credential.|
+|`CredentialType`|✔️|`string`|`Blob`|A (hex-encoded) value to identify the type of credential from the issuer.|
 |`Expiration`| |`number`|`UInt32`|Optional credential expiration.|
 |`URI`| |`string`|`Blob`|Optional additional data about the credential (such as a link to the VC document). This field isn't checked for validity and is limited to a maximum length of 256 bytes.|
-|`OwnerNode`|✔️|`string`|`UInt64`|A hint indicating which page of the owner's owner directory links to this object, in case the directory consists of multiple pages. Note: The object does not contain a direct link to the owner directory containing it, since that value can be derived from the `Account.PreviousTxnID`.|
+|`SubjectNode`|✔️|`string`|`UInt64`|A hint indicating which page of the subject's owner directory links to this object, in case the directory consists of multiple pages.|
+|`IssuerNode`|✔️|`string`|`UInt64`|A hint indicating which page of the issuer's owner directory links to this object, in case the directory consists of multiple pages.|
```

Comment:
That becomes confusing, because the "Owner" could be either the issuer or the subject, depending on whether the credential is created or not. The pointer to the subject's owner directory should be there even if the subject hasn't accepted it.

Reply:
What do you mean by trustline ? I don't see any trustline ledger object. 

You can see other objects like Escrow, Paychan etc. They all have sfOwnerNode and other object pointer like sfDestinationNode.  

And why non-accepted credential should have entry in the subject directory? May be subject don't want to be associated with this entry?

Reply:
> What do you mean by trustline ? I don't see any trustline ledger object.

The object name is `RippleState`.

> And why non-accepted credential should have entry in the subject directory? May be subject don't want to be associated with this entry?

They can delete it if they want. The same is true of an escrow's destination.